feature-libs/organization/administration/components/unit/list/toggle-link/toggle-link-cell.component.html

```diff
@@ -1,5 +1,6 @@
 <a
   *ngIf="hasItem"
+  class="invisible-focus-border"
```

Comment:
change the name to `hide-focus-border` or something similar. invisible isn't the best word choice here

Reply:
I got your point. "Invisible" normally means the element is still there but cannot be observed by end user, "hide/hiden" means it's physically gone. I agree that "hide" is better than "invisible".